### ISSUE TITLE
return string without instantiate var

### ISSUE THREAD

**Issue by Cedric-ruiu** [jenkey2011/string-highlight#1]
Hi,

It doesn't work properly when it doesn't store string template into variable.

```javascript
// work
const tpl = `<p>Yep</p>`;

// doesn't work
return `<p>Yep</p>`;
```

Cool extension.

**Comment by jenkey2011** [comment]
Fix in v0.0.8. https://marketplace.visualstudio.com/items?itemName=Jenkey2011.string-highlight
Thanks for the compliment

**Comment by Cedric-ruiu** [comment]
Fast !

Tested & Fixed thx ;)
